src/app/app.component.html

```diff
@@ -3,4 +3,51 @@ <h1>
     Najlepsza {{ title }} w sieci!
   </h1>
   <img width="300" alt="Angular Logo" src="http://vitaltrading.pl/wp-content/uploads/2016/04/checklist-1000x675.jpg">
-</div>
\ No newline at end of file
+</div>
```

Comment:
Do tego diva wrzuciłeś style w HTML. Nie rób tak, jeśli musisz je do niego zastosować rób to zawsze w CSS i nadawaj tego typu elementom klasy.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/1c8cc3e6fd1c5f56d6931b216180d2b43c17acd1